### PR TITLE
Add twirling options to Shots description

### LINKED ISSUE #1857: Mention twirling options `num_randomizations` and `shots_per_randomization` in shots section
### URL, if applicable

https://docs.quantum.ibm.com/guides/specify-runtime-options#shots-sampler-only

### Describe the new content you are requesting.

In the `shots` and `precision` section we should talk about the twirling options `num_randomizations` and `shots_per_randomization` because they can effect the number of shots used. 

If shots are passed in as an option `estimator.options.default_shots = shots` then the value passed in will be altered. This behavior is briefly mentioned in https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.TwirlingOptions

### If this new content request is accepted, do you want to write the content?

I can help the team by providing enough information to write the material

### PR DESCRIPTION
**Pull request by beckykd**
closes #1857

### CODE REVIEW

File: docs/guides/specify-runtime-options.mdx [hunk 1]
```diff
@@ -138,10 +138,15 @@ estimator.options.resilience.zne_mitigation = True
...
+However, if `shots` is not specified by any PUB or in the run keyword argument ( or if they are all `None`), then the shots value from the options is used, most notably `default_shots`.
+
+Finally, if the `twirling` options `num_randomizations` or `shots_per_randomization` are specified, those values take precedence over `default_shots`  (but not `shots` values specified as part of the `run` call).
+
+To summarize, this is the order of precedence for specifying shots in the Sampler, for any particular PUB:
 
 1. If the PUB specifies shots, use that value.
-2. If the shots keyword argument is specified in `run`, use that value.
+2. If the `shots` keyword argument is specified in `run`, use that value.
+3. If `num_randomizations` or `shots_per_randomization` are specified as twirling options, use those values.
```
Comment (kt474) [inline docs/guides/specify-runtime-options.mdx:149]: I would reword it to specify that these `twirling` options are enabled by default so if `default_shots` are the only way shots are set then the number of shots will be:

the product of ``num_randomizations`` and ``shots_per_randomization``

Maybe we can also link the twirling options (https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.TwirlingOptions) - Jessie's new updates (https://github.com/Qiskit/qiskit-ibm-runtime/pull/1884) aren't live yet but will be on next release

**Review by kt474 — APPROVED** [review]

**Comment by Eric-Arellano** [comment]
@abbycross Becky is OOO. Can you please review this and merge once ready?

**Comment by abbycross** [comment]
> @abbycross Becky is OOO. Can you please review this and merge once ready?

Yep no problem, she pinged me about it -- will do!

File: docs/guides/specify-runtime-options.mdx [hunk 2]
```diff
@@ -138,20 +138,28 @@ estimator.options.resilience.zne_mitigation = True
...
 
-However, if `shots` is not specified by any PUB or in the run keyword argument (equivalently, if they are all `None`), then the shots value from the options is used, most notably `default_shots`. To summarize, this is the order of precedence for specifying shots in the Sampler, for any particular PUB:
+However, if `shots` is not specified by any PUB or in the run keyword argument (or if they are all `None`), then the shots value from the options is used, most notably `default_shots`.
+
+Finally, because the [`twirling` options](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.TwirlingOptions) `num_randomizations` and `shots_per_randomization` are enabled by default, if the only way shots are specified is by the `default_shots` value, the number of shots will actually be the product of `num_randomizations` and `shots_per_randomization`.
```
Comment (abbycross) [inline docs/guides/specify-runtime-options.mdx:143]:
```suggestion
Finally, because the [`twirling` options](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.TwirlingOptions) `num_randomizations` and `shots_per_randomization` are enabled by default, the number of shots will actually be the product of `num_randomizations` and `shots_per_randomization` if the `default_shots` value is the only way shots are specified.
```

File: docs/guides/specify-runtime-options.mdx [hunk 3]
```diff
@@ -138,20 +138,28 @@ estimator.options.resilience.zne_mitigation = True
...
+3. If `num_randomizations` and `shots_per_randomization` are specified as `twirling` options (enabled by default), shots are the product of those values.
 3. If `sampler.options.default_shots` is specified, use that value.
 
-For example, if shots are specified in all three places, the one with highest precedence (shots specified in the PUB) is used.
+Thus, if shots are specified in all possible places, the one with highest precedence (shots specified in the PUB) is used.
 
 #### Precision (Estimator only)
 
-Precision is analogous to shots, described in the previous section, except that the Estimator options contains both `default_shots` and `default_precision`. Specifically, for any particular Estimator PUB:
+Precision is analogous to shots, described in the previous section, except that the Estimator options contains both `default_shots` and `default_precision`.
```
Comment (abbycross) [inline docs/guides/specify-runtime-options.mdx:156]:
```suggestion
Precision is analogous to shots, described in the previous section, except that the Estimator options contain both `default_shots` and `default_precision`.
```

**Review by abbycross — APPROVED** [review]
